mp/src/game/server/neo/neo_player.cpp

```diff
@@ -2533,7 +2533,7 @@ void CNEO_Player::GiveLoadoutWeapon(void)
 {
 	const NeoRoundStatus status = NEORules()->GetRoundStatus();
 	if (!(status == NeoRoundStatus::Idle || status == NeoRoundStatus::Warmup) && 
-		(IsObserver() || IsDead() || m_bDroppedAnything || ((NEORules()->GetRemainingPreRoundFreezeTime(false)) < 0)))
+		(IsObserver() || IsDead() || m_bIneligibleForLoadoutPick || m_aliveTimer.IsGreaterThen(25.f)))
```

Comment:
Is it fine to hardcode this value?

Reply:
Are you complaining about magic numbers or saying I should add a convar

Reply:
Magic number - yes, convar - dunno, maybe? We need an another opinion.

Reply:
Probably would be nice as a convar, but personally I don't see a use for it in any scenario.

Reply:
in some other game modes might be worth changing this settings, Ill make it a convar